src/Calculator/Views/GraphingCalculator/GraphingCalculator.xaml.cpp

```diff
@@ -159,7 +160,12 @@ void GraphingCalculator::OnEquationsVectorChanged(IObservableVector<EquationView
 
 void GraphingCalculator::OnTracePointChanged(Windows::Foundation::Point newPoint)
 {
-    TraceValue->Text = "(" + newPoint.X.ToString() + ", " + newPoint.Y.ToString() + ")";
+    wstringstream traceValueString;
+
+    // TODO: The below precision should ideally be dynamic based on the current scale of the graph.
+    traceValueString << "(" << fixed << setprecision(1) << newPoint.X << "," << fixed << setprecision(1) << newPoint.Y << ")";
```

Comment:
NIT: the space character is missing

`<< ", " <<`

Reply:
Suggestion: should we display something like `x=43, y=32` instead of `(43, 32)`?

cc @grochocki

Reply:
Done